bower.json

```diff
@@ -25,7 +25,11 @@
     "paper-radio-button": "PolymerElements/paper-radio-button#^2.0.0",
     "paper-tabs": "PolymerElements/paper-tabs#^2.0.0",
     "paper-toggle-button": "PolymerElements/paper-toggle-button#^2.0.0",
-    "polymer": "Polymer/polymer#^2.0.0"
+    "polymer": "Polymer/polymer#^2.0.1"
```

Comment:
Discussed with Nicolas. I should have git clean before. The bower.json changes are not necessary.